xarray/core/dataset.py

```diff
@@ -1815,24 +1815,29 @@ def reindex(self, indexers=None, method=None, tolerance=None, copy=True,
         return self._replace_vars_and_dims(variables, coord_names)
 
     def interp(self, coords=None, method='linear', assume_sorted=False,
-               kwargs={}, **coords_kwargs):
+               keep_attrs=False, kwargs={}, **coords_kwargs):
```

Comment:
My inclination would be to have `keep_attrs=True` by default, and maybe not even bother to include the argument at all. This would be consistent with `reindex`/`sel`/`isel`.

Reply:
OK. Then, I would remove this argument.